addBag.php

```diff
@@ -1,16 +1,22 @@
+
+<?php
+    require_once ('getDbFunction.php');
+    require_once ('checkImageUrlFunction.php');
+    require_once ('checkEntriesFunction.php');
+    require_once ('addBagFunction.php');
+?>
 <html lang=en>
 <head>
         <link rel="stylesheet" type="text/css" href="normalize.css">
         <link rel="stylesheet" type="text/css" href="style.css">
-        <link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
         <link href="https://fonts.googleapis.com/css?family=Questrial&display=swap" rel="stylesheet">
         <title>My Collection</title>
     </head>
     <body>
         <h1>My Handbag Collection</h1>
         <nav>
             <div class="navbar">
-                <a class="underline" href="index.php">Collection</a>
+                <a class="hover" href="index.php">Collection</a>
                 <a class="underline">Add Item</a>
```

Comment:
Is there a reason why you're doing it this way rather than a class="underline hover"?